test/e2e_node/inode_eviction_test.go

```diff
@@ -93,10 +93,13 @@ var _ = framework.KubeDescribe("InodeEviction [Slow] [Serial] [Disruptive] [Flak
 	}
 	evictionTestTimeout := 30 * time.Minute
 	testCondition := "Disk Pressure due to Inodes"
+	inodesConsumed := uint64(200000)
 
 	Context(fmt.Sprintf("when we run containers that should cause %s", testCondition), func() {
 		tempSetCurrentKubeletConfig(f, func(initialConfig *kubeletconfig.KubeletConfiguration) {
-			initialConfig.EvictionHard = "nodefs.inodesFree<70%"
+			// Set the eviction threshold to inodesFree - inodesConsumed, so that using inodesConsumed causes an eviction.
+			initialConfig.EvictionHard = fmt.Sprintf("nodefs.inodesFree<%d", getInodesFree()-inodesConsumed)
+			initialConfig.EvictionMinimumReclaim = ""
```

Comment:
I want to make this test more consistent.  Minimum reclaim is designed to evict extra pods, even if we have eliminated disk pressure.  If we are testing the ability of the kubelet to evict only pods that are abusive consistently, then this is an unnecessary complication (that we probably can't consistently pass).